src/definitions/modules/dropdown.less

```diff
@@ -434,7 +434,6 @@ select.ui.dropdown {
         overflow-y: auto;
         overscroll-behavior: @overscrollBehavior;
         backface-visibility: hidden;
-        -webkit-overflow-scrolling: touch;
```

Comment:
https://caniuse.com/mdn-css_properties_-webkit-overflow-scrolling

supported in Safari until 2019, then no support in any browser

Reply:
Since 2.9.0 we stated on the readme to only support the last 4 versions of ios Safari..That is 13-16, so ok for me